src/docs/product/integrations/integration-platform/ui-components/formfield.mdx

```diff
@@ -0,0 +1,109 @@
+---
+title: FormField Components
+sidebar_order: 4
+description: "Learn about FormField components."
+---
+
+The issue linking and alert rule action UI components can utilize `FormField` components to display attributes in their respective modals. Some UI Components (e.g. [Issue Link](/product/integrations/integration-platform/ui-components/issue-link), [Alert Rule Actions](/product/integrations/integration-platform/ui-components/alert-rule-action)) can render custom forms with inputs specified according to the schemes below. They behave similarly to their HTML equivalents.
```

Comment:
Streamlined this because it was repetitive

```suggestion
The [issue linking](/product/integrations/integration-platform/ui-components/issue-link) and [alert rule action](/product/integrations/integration-platform/ui-components/alert-rule-action) UI components can use `FormField` components to display attributes in their respective modals. These components can render custom forms with inputs specified according to the schemes below. They behave similarly to their HTML equivalents.
```